From 669261330f1b7e63c571c4b0811e0b979b4a8c10 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 10 Apr 2021 20:13:10 +0200
Subject: [PATCH] doc: imx: psb: Footnote not referenced

The current text creates build warnings when running 'make htmldocs'.

doc/imx/misc/psb.rst:173: WARNING: Footnote [1] is not referenced.
doc/imx/misc/psb.rst:174: WARNING: Footnote [2] is not referenced.

Avoid the build warning by converting the footnotes to list items.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/imx/misc/psb.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/imx/misc/psb.rst b/doc/imx/misc/psb.rst
index 4d6f001b74..98b419f138 100644
--- a/doc/imx/misc/psb.rst
+++ b/doc/imx/misc/psb.rst
@@ -170,5 +170,5 @@ A WARM reset can be triggered using WDT as follows::
 References
 ----------
 
-.. [1] i.MX 7Dual Applications Processor Reference Manual, Rev. 1, 01/2018 ; section 6.6.5.3.5 Redundant boot support for expansion device
-.. [2] i.MX 8M Mini Applications Processor Reference Manual, Rev. 3, 11/2020 ; section 6.1.5.4.5 Redundant boot support for expansion device
+* i.MX 7Dual Applications Processor Reference Manual, Rev. 1, 01/2018 ; section 6.6.5.3.5 Redundant boot support for expansion device
+* i.MX 8M Mini Applications Processor Reference Manual, Rev. 3, 11/2020 ; section 6.1.5.4.5 Redundant boot support for expansion device
-- 
2.30.2

